presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -127,6 +129,22 @@ public boolean run()
             }
         }
 
+        // Read queries from stdin
+        if (!hasQuery && !detectRealTerminal()) {
+            try {
+                if (System.in.available() > 0) {
+                    query = new String(ByteStreams.toByteArray(System.in), terminal().encoding()) + ";";
```

Comment:
are we chopping this into indivual queries later?

Reply:
Yeah, it's being done in the non-changed code path which was working before for `--execute` and `-f`